src/lib/receiver.js

```diff
@@ -26,6 +26,12 @@ const PSK_RECEIVER_ID_PREFIX = '~psk.'
  */
 
 /**
+ * @typedef {Object} PSKParams
```

Comment:
Should probably be `PskParams` - not saying the code is currently consistent, but I'm seeing more HttpRequest than HTTPRequest.